### PR TITLE
Add BACE integration

### PR DESCRIPTION
**Pull request by arianmaghsoudnia**

### CODE REVIEW

File: groups/bace-iot-platform/measurements/config.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+version: 1
+name: Measurements
```
Comment (royyeah) [inline groups/bace-iot-platform/measurements/config.yml:2]: I would suggest to change the folder to `measurements-webhook` and the name to 'Measurements webhook'

File: groups/bace-iot-platform/index.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+name: BACE IoT platform
```
Comment (royyeah) [inline groups/bace-iot-platform/index.yml:1]: Because it is a product title I would write 'BACE IoT Platform' with the last word capitalized as well.

**Review by royyeah — COMMENTED** [review]
@arianmaghsoudnia I add some comments from my side, I think @arjanvanijzendoorn can best decide who can review the rest

File: groups/bace-iot-platform/measurements-webhook/payloads/testPayload.json [hunk 3]
```diff
@@ -0,0 +1,118 @@
+[
```
Comment (arjanvanijzendoorn) [inline groups/bace-iot-platform/measurements-webhook/payloads/testPayload.json:1]: Can you reformat this file with Visual Studio Code (or something) to fix indentation and make sure there is a newline at the end.

File: groups/bace-iot-platform/measurements-webhook/tests.yml [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    expectedMeasurements:
+        - ingestionId: f1df36e6-05d9-4997-9d70-ca09d2d8695d$temperature
+          value: 28.84399414
+          date: 2023-01-01T00:00:00.000+00:00
+        - ingestionId: f1df36e6-05d9-4997-9d70-ca09d2d8695d$relative-humidity
+          value: 32.65380859
+          date: 2023-01-01T00:00:00.000+00:00
+        - ingestionId: f1df36e6-05d9-4997-9d70-ca09d2d8695d$light-intensity
+          value: 0.07
+          date: 2023-01-01T00:00:00.000+00:00
```
Comment (arjanvanijzendoorn) [inline groups/bace-iot-platform/measurements-webhook/tests.yml:13]: Maybe add a test that shows that no measurements are created if required fields are missing?
And a newline at the end of this file 😉

**Review by arjanvanijzendoorn — CHANGES_REQUESTED** [review]
Reviewed 3 commits. 2 comments.

**Pull request merged**